### PR TITLE
fix: add EPL 1.0 mapping

### PR DESCRIPTION
**Pull request by iseki0**
Fixes: #15

### CODE REVIEW

**Comment by chrimaeon** [comment]
For this to get merged:

1. change the branch to commit to to `develop`
2. add the actual license text to `src/main/resources/licenses`
3. add the SPDX Identifier mapping to `com.cmgapps.license.model.LicenseId`
4. add the file mapping to `com.cmgapps.license.helper.LicensesHelperKt#getFilename`

To check if your changes work you should add a test that uses your new license.

**Review by chrimaeon — CHANGES_REQUESTED** [review]
For this to get merged:

1. change the branch to commit to to develop
2. add the actual license text to src/main/resources/licenses
3. add the SPDX Identifier mapping to com.cmgapps.license.model.LicenseId
4. add the file mapping to com.cmgapps.license.helper.LicensesHelperKt#getFilename

To check if your changes work you should add a test that uses your new license.

File: src/main/resources/license_map.csv [hunk 1]
```diff
@@ -21,6 +21,7 @@ http://opensource.org/licenses/BSD-3-Clause,BSD_3
 https://opensource.org/licenses/BSD-3-Clause,BSD_3
 EPL-2.0,EPL_2
 Eclipse Public License 2.0,EPL_2
+Eclipse Public License - v 1.0,EPL_1
```
Comment (chrimaeon) [inline src/main/resources/license_map.csv:24]: Please also add the url `http://www.eclipse.org/legal/epl-v10.html` and not put it in between the EPL_2's. Add it at the end of the file

Comment (iseki0) [reply src/main/resources/license_map.csv:24]: OK, I will re-create a PR for it

Comment (iseki0) [reply src/main/resources/license_map.csv:24]: OK, I will re-create a PR for it

**Pull request closed without merging**